### DISCUSSION TITLE
The list of contributors needs to be updated.

### DISCUSSION THREAD

**Discussion by setanarut** [iina/iina#3678] [Show and tell]
The translations are done with many contributors via Crowdin. But there are some old special names in the About section. I translated almost all Turkish. but there is one person on the list from the old first translation. Since there are multiple translators, that list needs to be updated. This goes against the spirit of open source.

<img width="612" alt="ekran" src="https://user-images.githubusercontent.com/36481442/161311289-b42ad5e5-ba51-40c2-85fc-7f0cff3a9f39.png">

**Comment by saagarjha** [comment]
I think https://github.com/iina/iina/blob/develop/iina/Translators.json is the file that keeps track of this, so feel free to add yourself to it :)

**Reply by setanarut** [reply]
You cannot do this manually. Crowdin is a platform where multiple people translate simultaneously. You can view the activity [here](https://crowdin.com/project/iina/activity-stream). I think all translators should be thanked at once without mentioning their names.

**Reply by setanarut** [reply]
or a Markdown page can be created to listing all the names in the Crowdin.
